### ISSUE TITLE
[Documentation] Webserver user needs to be in the docker group

### ISSUE THREAD

**Issue by Dungeonseeker** [pterodactyl/panel#4692]
### Current Behavior

Webpanel cannot connect to containerised servers unless webserver user is in the docker group.

Might we worthwhile updating the documentation to include

sudo usermod -aG docker www-data (Ubuntu)
sudo usermod -aG docker apache/nginx (CentOS)

### Expected Behavior

N/A

### Steps to Reproduce

Follow guide exactly as it is laid out and the result is a webpage that refuses to connect to servers at all, it will install them fine but as soon as they finish installing it changes to "problems connecting to server" and servers don't actually start/stop.

As soon as i added www-data user to docker group it worked instantaneously.

### Panel Version

1.4.0

### Wings Version

1.4.0

### Games and/or Eggs Affected

all

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by TrixterTheTux** [comment]
The panel doesn't touch Docker in any way (and this is an incredibly dangerous configuration), chances are the issue is something else.

**Comment by QuintenQVD0** [comment]
and your panel and wings version is old

**Comment by Dungeonseeker** [comment]
> and your panel and wings version is old

apologies, I somehow managed to type the entirely wrong thing twice (think I was typing back what it already said, it was early and I hadn't had any coffee yet)

Panel - 1.11.3
Wings - 1.11.4

> The panel doesn't touch Docker in any way (and this is an incredibly dangerous configuration), chances are the issue is something else.

Kinda weird how it fixed my issue if its entirely unrelated though, right?

I'm gonna remove www-data from the docker group and see if it breaks again.

EDIT - Well you are indeed correct, panel seems to work fine now even without the group. I have zero idea what I did to fix it though, I spent like 3 hours yesterday and the panel simply refused to connect to a server. TBC I could talk to the node and create servers, they would install and I could browse the files with sftp but when ever I went to the console page a red bar would appear at the top saying "Unable to connect to server" and the console was just blank.
